packages/components/src/List/ItemTitle/__snapshots__/ItemTitle.test.js.snap

```diff
@@ -0,0 +1,115 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ItemTitle should render button title 1`] = `
+<div
+  className={undefined}
```

Comment:
the className is undefined because we use theme['item-title'] sass and in tests it's mocked.